src/FsAutoComplete/FsAutoComplete.Lsp.fs

```diff
@@ -461,8 +461,8 @@ type FSharpLspServer(backgroundServiceEnabled: bool, state: State, lspClient: FS
 
           diagnosticCollections.SetFor(uri, "F# Analyzers", diags)
       | NotificationEvent.TestDetected(file, tests) ->
-          let res = CommandResponse.test JsonSerializer.writeJson (UMX.untag file) tests
-          { Content = res }
+          { File = UMX.untag file
```

Comment:
another minor change I'd like: when communicating with an LSP, instead of returning 'raw' file paths like this we should return file URIs, as this is the structure the LSP protocol itself uses for reasoning about files.  this can be done easily with `Path.LocalPathToUri: string<LocalPath> -> string`, which should be in scope here.